content/release.rst

```diff
@@ -485,10 +516,8 @@ any time.
 Especially if a security bugfix is found, we will make a new release as soon
 as the bug is fixed.
 
-**Other questions to ask:**
-
-* If a customer-requested feature is added or if a bug fix is made, who
-  should be contacted to notify them of the new release?
-
-* Are there further updates for non-Documentation related content, like the
-  website or newsletter?
+Releases may include bugfixes or product features from customers.
```

Comment:
Maybe: may include fixed for defects observed by customers or new product features requested by customers.